### PR TITLE
Add comment issues8

### PR DESCRIPTION
**Pull request by Msdot001**
# Description

Create a Comment model and test the model using a unit test 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Msdot001** [comment]
creation and testing of a comment model.

**Review by aibunny — DISMISSED** [review]

**Review by Barissa-Imran — CHANGES_REQUESTED** [review]
From the meeting, we agreed that we are to use the pytest module for running our tests. Please see: 
- [Pytest](https://pytest.org/)

How we have used it before:
- [account tests](https://github.com/SpaceyaTech/blog/blob/main/CMS/tests/accounts/test_account_serializers.py)

Also check for any conflicts you might be getting from this pr

File: blog/admin.py [hunk 1]
```diff
@@ -1,13 +1,24 @@
 from django.contrib import admin
-from blog.models import Stori,Category ,Comment
+
+from blog.models import Stori,Category, Comment
+
+
+class CommentInline(admin.TabularInline):     
+    model = Comment
+    extra = 0
+
```
Comment (JimmyTron) [inline blog/admin.py:9]: I see it

**Review by JimmyTron — APPROVED** [review]

**Comment by Msdot001** [comment]
Okay.

I will work on it. Thanks

**Review by IanoNjuguna — DISMISSED** [review]

**Review by IanoNjuguna — APPROVED** [review]

**Review by JimmyTron — APPROVED** [review]

**Pull request closed without merging**